static/app/components/slideOverPanel.tsx

```diff
@@ -59,6 +59,9 @@ function SlideOverPanel(
             stiffness: 500,
             damping: 50,
           }}
+          role="complementary"
+          aria-hidden={collapsed}
+          aria-label="slide-out-drawer"
```

Comment:
This should be a human-readable label. Screen readers will announce the label whit the panel appears